### ISSUE TITLE
Note on compile error

### ISSUE THREAD

**Issue by jephthai** [lfe/lfe#175]
I ran into this tonight while compiling LFE, and thought you might be able to enhance the documentation a bit.  I encountered this issue after cloning the repo and attempting a compile, following the build instructions in the README:

```
$ make compile
escript get_maps_opts.escript
if which rebar.cmd > /dev/null; \
    then ERL_LIBS=.:$ERL_LIBS rebar.cmd compile; \
    elif which rebar > /dev/null; \
    then ERL_LIBS=.:$ERL_LIBS rebar compile; \
    else \
    make  erlc-lfec; \
    fi
make[1]: Entering directory `/home/josh/tools/lfe'
make[1]: *** No rule to make target `ebin/lfe_bits.beam', needed by `erlc-compile'.  Stop.
make[1]: Leaving directory `/home/josh/tools/lfe'
make: *** [compile] Error 2
```

The message makes it look like there was some error in the Makefile.  I'm not a Makefile savant by any means, but after playing around for a bit, I realized that I have Erlang installed on my Ubuntu box, but not rebar.  Once I did an "apt-get install rebar", the compile worked great. 

It occurred to me that the dependency on rebar could be more clearly indicated in the build instructions in the README -- it currently suggests that "erl" is the only dependency.

Thanks!

**Comment by rvirding** [comment]
Which version are you using? A bug has crept into the latest version in the develop branch in Makefile which should be fixed:

```
diff --git a/Makefile b/Makefile
index c113a47..963cf93 100644
--- a/Makefile
+++ b/Makefile
@@ -54,7 +54,7 @@ DESTBINDIR = $(PREFIX)$(shell dirname `which erl` 2> /dev/null
 $(BINDIR)/%: $(CSRCDIR)/%.c
        cc -o $@ $<

-$(EBINDIR)/%.beam: $(EBINDIR) $(SRCDIR)/%.erl
+$(EBINDIR)/%.beam: $(SRCDIR)/%.erl
        @mkdir -p $(EBINDIR)
        $(ERLC) -I $(INCDIR) -o $(EBINDIR) $(MAPS_OPTS) $(ERLCFLAGS) $<
```

Apart from that it seems to work ok for me. You don't/shouldn't need rebar, though if it can find it then it uses it. I am not a rebar person myself so I tend to explicitly bypass it.

**Comment by jephthai** [comment]
I checked out the git repo about half an hour before I posted my message above.  My "VERSION" file says 0.11.0-dev.

I never use rebar at all (hence the fact that I didn't have it installed on this machine!), but it is definitely the case that installing rebar got around the error message above. 

At any rate, the diff you provided there works for me -- removing the extra `$(EBINDIR)` does the trick.

Thanks!
